pages/index.tsx

```diff
@@ -116,7 +116,18 @@ const Home: NextPage = () => {
         <div className="header">
           <img src="/images/livekit-meet-home.svg" alt="LiveKit Meet" width="360" height="45" />
           <h2>
-            Open source video conferencing app built on LiveKit&nbsp;Components, LiveKit&nbsp;Cloud,
+            Open source video conferencing app built on{' '}
+            <a
+              href="https://github.com/livekit/components-js?ref=meet"
+              target="_blank"
+              rel="noreferrer"
```

Comment:
in order to track links here, it might actually make sense to remove `rel="noreferrer"` everywhere as we might potentially want to use this insight. 
we could replace it with `rel="noopener"` for security.

Reply:
Apparently `target="_blank"` with `rel="noopener"` is a security risk for older browsers. You need to remove the `target="_blank"` to prevent this.
TIL: Using target="_blank" is bad practice for most use cases. [Source](https://css-tricks.com/use-target_blank/)